websocket-client-simple.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_development_dependency "bundler", "~> 1.3"
-  spec.add_development_dependency "rake"
-  spec.add_development_dependency "minitest"
-  spec.add_development_dependency "websocket-eventmachine-server"
-  spec.add_development_dependency "eventmachine"
+  spec.add_development_dependency "rake", "~> 12.3.2"
+  spec.add_development_dependency "minitest", "~> 5.11.3"
+  spec.add_development_dependency "websocket-eventmachine-server", ">= 0"
+  spec.add_development_dependency "eventmachine", "~> 1.2.7"
 
-  spec.add_dependency "websocket"
-  spec.add_dependency "event_emitter"
+  spec.add_dependency "websocket", "~> 1.2.8"
+  spec.add_dependency "event_emitter", "~> 0.2.6"
```

Comment:
Why did you edit those lines? It does not affect actually and not breaking, also not needed.